deny.toml

```diff
@@ -0,0 +1,25 @@
+[advisories]
+ignore = [
+  "RUSTSEC-2024-0436", # Paste is unmaintained, whatever.
+]
+
+[bans]
+multiple-versions = "allow"
+
+[sources]
+allow-git = [ "https://github.com/filecoin-project/builtin-actors" ]
+
+[licenses]
+allow = [
+  "Apache-2.0",
+  "Apache-2.0 WITH LLVM-exception",
+  "BSD-2-Clause",
+  "BSD-3-Clause",
+  "CC0-1.0",
+  "MIT",
+  "NCSA",
+  "Unlicense",
+  "MPL-2.0",
+  "Unicode-3.0",
+  "Zlib",
```

Comment:
These are all "permissive" licenses. But we should try to trim down our dependency tree a bit.